src/components/menu/menu-trigger/menu-trigger.directive.ts

```diff
@@ -33,7 +34,12 @@ export class MenuTriggerDirective implements OnInit, OnDestroy {
     private _portal: TemplatePortal;
 
     /** Store the reference to the overlay */
-    private _overlayRef: OverlayRef;
+    private _overlayRef?: OverlayRef;
+
+    /** Get the aria controls for accessibility */
+    get ariaControls(): string | null {
+        return this.menu?.isMenuOpen ? this.menu?.id : null;
```

Comment:
aria-controls needs to refer to the id of the element with `role="menu"`, currently it refer to this empty element:
![image](https://user-images.githubusercontent.com/26114101/126146474-d5bdcbb2-3e34-48f1-b1fa-aa3d7c124445.png)

Reply:
Thanks for catching that Donna, just as a side note, should the `role="menu"` be removed from that empty ux-menu element?